### PR TITLE
MV-1949 updated message-delivered callback doc

### PR DESCRIPTION
**Pull request by arifibrahim4**

### CODE REVIEW

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-MV-1949-Update-Message-Delivered-Callback-doc.s3-website-us-east-1.amazonaws.com/

File: messaging/callbacks/msgDelivered.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ MMS and Group messages <b>don’t</b> currently support delivery receipts. Howev
...
 | message.from          | `string` | The phone number the message was sent from                                                                                                                                                                                                                                                                                                                          |
-| message.text          | `string` | The text content of the message                                                                                                                                                                                                                                                                                                                                     |
+| message.text          | `string` | Empty text/string.                                                                                                                                                                                                                                                                                                                                    |
```
Comment (jmulford-bw) [inline messaging/callbacks/msgDelivered.md]: Maybe add a small comment as to why we're having this be an empty string. Something like `Bandwidth no longer text contents for message delivered callbacks`?

**Review by jmulford-bw — APPROVED** [review]

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-MV-1949-Update-Message-Delivered-Callback-doc.s3-website-us-east-1.amazonaws.com/

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-MV-1949-Update-Message-Delivered-Callback-doc.s3-website-us-east-1.amazonaws.com/

**Review by clink-bw — APPROVED** [review]

**Pull request merged**